### DISCUSSION TITLE
explorer.exe not terminated by SEB automatically

### DISCUSSION THREAD

**Discussion by Chris251290** [SafeExamBrowser/seb-win-refactoring#469] [Q&A]
Hi Team,

We have given "**explorer.exe**" on the Prohibited process in SEB Configuration. 
But, While launching SEB, it asks to terminate "**explorer.exe**" manually. (All other applications in the prohibited process are terminated automatically.)

Here I attached the screenshots. Please check and give me a solution.
![Screenshot from 2022-09-16 16-39-22](https://user-images.githubusercontent.com/111885140/191023831-52786173-1b9f-47ba-b5f9-3a677d9a388d.png)

![Screenshot from 2022-09-16 16-24-19](https://user-images.githubusercontent.com/111885140/191023834-e6cb1be3-289b-4190-9d32-955eabad62f9.png)

**Comment by dbuechel** [comment]
There is no need to put `explorer.exe` in the list of prohibited processes, it is automatically terminated and/or monitored depending upon which kiosk mode you're using.

**Reply by Chris251290** [reply]
Thanks for your reply Damian.

So, Do I need to enable "**Monitor processes while SEB is running**"? If yes, How to enable it?

One more query..

If I permit any applications like pdf, excel, Word & Notepad in the permitted processess, Do I need to put the "explorer.exe" also in the permitted processes?

**Reply by dbuechel** [reply]
No, you don't need to add `explorer.exe` to the permitted applications.

**Reply by dbuechel** [reply]
> Do I need to enable "Monitor processes while SEB is running"? If yes, How to enable it?

It is automatically enabled when a kiosk mode is active.

**Reply by Chris251290** [reply]
Thank you Damian...

Same issue arise while we prohibit "**AnyDesk.exe**" in the prohibited processes. SEB asked to terminate Anydesk manually.

Could you please tell the solution for this? Because we need to block the Anydesk application.

**Reply by dbuechel** [reply]
https://github.com/SafeExamBrowser/seb-win-refactoring/discussions/451#discussioncomment-3475319
